download_models/downloadOVMSModels.sh

```diff
@@ -93,7 +93,18 @@ downloadOMZmodel(){
 
     (
         # create folder 1 under each precision FP directory to hold the .bin and .xml files
-        cd "$modelDir"/intel || { echo "Error: folder \"intel\" was not created by converter."; exit 1; }
+        omzModelDir=""
+        if [ -d "$modelDir/intel" ]; then
+            omzModelDir="$modelDir/intel"
+        elif [ -d "$modelDir/public" ]; then
+            omzModelDir="$modelDir/public"
+        else
+            echo "Error: folder \"$modelDir/intel\" or \"$modelDir/public\" was not created by converter."
+            exit 1
+        fi
+
+        cd "$omzModelDir" || { echo "Error: could not cd to folder \"$omzModelDir\"." ; exit 1; }
```

Comment:
i just realized that we should not do exit 1 in this function call and we should return 1 to do better error handling.